packages/api/src/hooks/useDocumentUpload.ts

```diff
@@ -55,26 +42,31 @@ const useDocumentUpload = () => {
 
             const updatedPayload = {
                 ...payload,
+                document_format: file.type
+                    .split('/')[1]
+                    .toLocaleUpperCase() as TDocumentUploadPayload['document_format'],
                 expected_checksum: checksum,
                 file_size: fileBuffer.length,
                 passthrough: {
                     document_upload: true,
                 },
             };
-            const chunks = generateChunks(fileBuffer, { chunkSize: 16384 });
-            setTempChunks(chunks);
-            await mutate({ payload: updatedPayload });
+            await mutateAsync({ payload: updatedPayload }).then(async res => {
```

Comment:
```suggestion
            setIsDocumentUploaded(false);
            await mutateAsync({ payload: updatedPayload }).then(async res => {
```
Probably best if we reset the uploaded status for multiple attempts